projtools/clean.py

```diff
@@ -1,30 +1,29 @@
 # instruction manual on how to use the data class stuff
-from data import Data # pylint: disable=no-name-in-module
+import pandas as pd
+from projtools.data import Data # pylint: disable=no-name-in-module
 
 data = Data()
 
 #renaming columns to make them more legible
-column_names = {"MajorText": "Category",
-                "MinorText": "Offence",
-                "LookUp_BoroughName": "Borough",
-                "Major Category": "Category",
-                "Minor Category": "Offence"}
-
-borough_10_21 = data.borough_crime_10_21
-borough_10_21 = borough_10_21.rename(columns=column_names)
-
-borough_21_23 = data.borough_crime_21_23
-borough_21_23 = borough_21_23.rename(columns=column_names)
-
-lsoa_10_21 = data.lsoa_crime_10_21
-lsoa_10_21 = lsoa_10_21.rename(columns=column_names)
-
-lsoa_21_23 = data.lsoa_crime_21_23
-lsoa_21_23 = lsoa_21_23.rename(columns=column_names)
-
-ward_10_21 = data.ward_crime_10_21
-ward_10_21 = ward_10_21.rename(columns=column_names)
-
-ward_21_23 = data.ward_crime_21_23
-ward_21_23 = ward_21_23.rename(columns=column_names)
-
+column_names = {"MajorText": "category",
+                "MinorText": "offence",
+                "LookUp_BoroughName": "borough",
+                "Major Category": "category",
+                "Minor Category": "offence",
+                "Borough": "borough",
+                "WardName": "ward_name",
+                "WardCode": "ward_code",
+                "LSOA Name": "lsoa_name",
+                "LSOA Code": "lsoa_code"}
+
+borough_10_21 = data.borough_10_21.rename(columns=column_names)
+borough_21_23 = data.borough_21_23.rename(columns=column_names)
+
+lsoa_10_21 = data.lsoa_10_21.rename(columns=column_names)
+lsoa_21_23 = data.lsoa_21_23.rename(columns=column_names)
+
+ward_10_21 = data.ward_10_21.rename(columns=column_names)
+ward_21_23 = data.ward_21_23.rename(columns=column_names)
+
+common_columns = ["ward_name", "ward_code", "category", "offence", "borough"]
+ward = pd.merge(ward_10_21, ward_21_23, on = common_columns, how = "outer")
```

Comment:
Does this work?

Reply:
How can I check it works?

Reply:
it works in that the code runs, but I can't be sure the data is correct

Reply:
Okay, so I'm gonna write some more tests but I'm confident the merge is good